modules/liftover/1.2/config/default.yaml

```diff
@@ -14,7 +14,11 @@ lcr-modules:
         inputs:
             sample_seg: "__UPDATE__"
             # Path to seg files for lifting. For example:
+            # For battenberg only:
             # sample_seg: "data/{tool}/{genome_build}/{tumour_sample_id}--{normal_sample_id}_subclones.igv.seg"
+            
+            # For Sequenza
+            # results/{tool}/03-igv_seg/{genome_build}/{tumour_sample_id}--{normal_sample_id}_--matched/filtered/{tool}_segments.igv.seg 
```

Comment:
Can you please update the example usage to point it to the 99-outputs?

Reply:
done